Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Definitions/Result.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel;
+
+namespace Frends.Excel.ConvertToJSON.Definitions;
+
+public class Result
+{
+    /// <summary>
+    /// False if conversion fails.
+    /// </summary>
+    [DefaultValue("false")]
+    public bool Success { get; set; }
+
+    /// <summary>
+    /// Exception message.
+    /// </summary>
+    [DefaultValue("")]
+    public string ErrorMessage { get; private set; }
```

Comment:
What is the reason for private set?

Reply:
Mostly for anyone not being able to overwrite. This way the value is read-only from outside (and the intention is that no one can set this value even by mistake from outside). Setter is still needed though for class private usage in ctor.